package.json

```diff
@@ -1,14 +1,15 @@
 {
   "name": "team-link",
   "version": "1.0.0",
-  "description": "Provides convenient interface to call all online team members via MS Teams",
-  "main": "src/index.js",
+  "description": "Provides a convenient interface to call all online team members via MS Teams",
+  "main": "src/index.ts",
```

Comment:
We shouldn't use TS files as an entrypoint. Your users won't be able to execute it without `ts-node`, it should point to the transpiled files. Ideally, we shouldn't distribute our TS sources at all, only transpiled JS files and generated `.d.ts` definition files.